src/pages/Explorer/Layouts/SheetsTemplate.svelte

```diff
@@ -0,0 +1,40 @@
+<script>
+    import Svg from 'webkit/ui/Svg/svelte'
+    export let item;
+    let isOpen = false;
+</script>
+
+<div class="column" on:click={() => isOpen = !isOpen}>
+    <div class="row justify v-center">
+        <h5 class="body-3">{item.title}</h5>
+        <Svg id='arrow' w="16" />
+    </div>
+    {#if isOpen}
+        <div class="row column mrg-l mrg--t mrg--b desc">
+            <p class="body-3 mrg-s mrg--b">
```

Comment:
`[D-1-1]` `body-3` rules are applied by default. It's needed only if the parent overwrote the default rules.